README.md

```diff
@@ -134,7 +134,8 @@ export OPTION=VALUE
 npm run build
 ```
 
-| Option                    | Example            | Description
-| ------------------------- | ------------------ | -----------
-| `PUBLIC_URL`              | `/studio`          | Path from which Studio will be served
-| `REACT_APP_SETTINGS_PATH` | `/mysettings.json` | Path from which to load the configuration
+| Option                            | Example            | Description
+| --------------------------------- | ------------------ | -----------
+| `PUBLIC_URL`                      | `/studio`          | Path from which Studio will be served
+| `REACT_APP_SETTINGS_PATH`         | `/mysettings.json` | Path from which to load the configuration
+| `REACT_APP_INCLUDE_LEGAL_NOTICES` | `1`                | If the legal notices should be included on the about page
```

Comment:
I would change the description a bit. Maybe:

> Set to `1` to include legal notices and information about ELAN e.V., any other value or having this variable not set will not include them. Unless you are working for ELAN e.V. there is probably no reason for you to use this variable.

This could certainly be improved but: it clearly says how this is probably not important for any "outside" users. Only "legal notices" without a mention of ELAN e.V. might make some people include it "because legal notices are always a good idea".